src/Teams/Models/TeamMembers.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Teams.Models
+{
+    public partial class TeamMembers
```

Comment:
why this class plural?
should be TeamMember, it is describe only one team member

Reply:
Fixed, sorry. Was already mentioned by you, my bad.

Reply:
also why it is partial ?

Reply:
Sorry, fixed that too. The original class has been scaffolded from Table (I used the DB first approach), and I did not carefully read through all of the auto-generated code before commiting. Will be more careful next time, thanks for noticing!